### PR TITLE
Several fixups

### PR DESCRIPTION
**Pull request by dnaranjo89**

### CODE REVIEW

File: server/forms/form_base.py [hunk 1]
```diff
@@ -31,8 +31,13 @@ class FormBase(forms.Form):
     def __init__(self, *args, **kwargs):
         super(FormBase, self).__init__(*args, **kwargs)
 
-        if 'initial' in kwargs and 'title' not in kwargs['initial']:
-            kwargs['initial']['title'] = self.DEFAULT_TITLE
+        if 'initial' in kwargs:
+            if 'title' not in kwargs['initial']:
+                kwargs['initial']['title'] = self.DEFAULT_TITLE
+
+            for attr in kwargs['initial']:
```
Comment (Mariocj89) [inline server/forms/form_base.py]: Pelease comment what is going here

File: server/forms/raffle_form.py [hunk 2]
```diff
@@ -23,12 +23,8 @@ class RaffleDrawForm(FormBase):
 
     def __init__(self, *args, **kwargs):
         if 'initial' in kwargs and 'participants' in kwargs['initial']:
-            kwargs['initial']['prices'] = ','.join(kwargs['initial']['prices'])
             if kwargs['initial']['registration_type'] == RaffleDraw.FACEBOOK:
-                participant_names = [u'{{{0}:{1}}}'.format(*participant_details) for participant_details in kwargs['initial']['participants']]
-            else:
-                participant_names = kwargs['initial']['participants']
-            kwargs['initial']['participants'] = ','.join(participant_names)
```
Comment (Mariocj89) [inline server/forms/raffle_form.py]: Explain what is going on. Does it work fine for restricted? I personally don't like the * operator for this kind of things

File: web/templates/index.html [hunk 3]
```diff
@@ -46,16 +46,16 @@
...
-            <input id='is-shared' type="checkbox">
-            <span>
-                <span>{% trans "Normal Draw" %}</span>
-                <span>{% trans "Shared Draw" %}</span>
-            </span>
-            <a></a>
-        </label>
-    </div>
+{#    <div title="" id="switch-draw-mode" class="col-sm-6 col-sm-offset-3">#}
+{#        <label class="switch-light switch-candy" >#}
```
Comment (Mariocj89) [inline web/templates/index.html]: Why don't remove it? Also the code for the tooltip and the warning to create a draw can be moved to create draw

Comment (dnaranjo89) [inline web/templates/index.html]: The is more css and js with it...
let's try how is the new flow going and later decide whether we finally remove it or not...

Comment (Mariocj89) [inline web/templates/index.html]: Fair

**Pull request merged**